sample/lit-todo/main.js

```diff
@@ -181,8 +181,15 @@ function update(newTodos) {
 window.addEventListener('load', () => {
   document.body.style.visibility = 'visible';
 });
+
 document.querySelector('mwc-fab').addEventListener('click', handleCreate);
 
+if ('serviceWorker' in navigator) {
```

Comment:
I'm pretty sure you just copied this from somewhere but the following also works:

```js
navigator.serviceWorker?.register('sw.js')
```

Reply:
Thanks!